content/guides/user-guides/delfin/driver-matrix-support/hp-3par-storeserv/_index.md

```diff
@@ -0,0 +1,54 @@
+---
+title: HP 3PAR StoreServ
+menuTitle: HP 3PAR StoreServ 
+description: "HP 3PAR StoreServ Support Matrix"
+weight: 20
+disableToc: false
+tags: ["user guide", "delfin", "drivers"]
+---
+
+### Model and versions supported
+
+ |Models   | Firmware Versions   |
```

Comment:
There seems to be some issue with the rendered table.
Please check the table.
![image](https://user-images.githubusercontent.com/19162717/107004599-89497900-67b4-11eb-8c6b-5f68a137a935.png)

Reply:
Rendering on my machine looks fine, PFA
![image](https://user-images.githubusercontent.com/48908765/107031105-acd1eb00-67d7-11eb-8556-89e942280429.png)